Ignore nonexistent system temporary directory when a valid cache directory is provided via options

bcremer opened the pull request:
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | yes
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | no

### Description

Fix for https://github.com/laminas/laminas-cache-storage-adapter-filesystem/issues/65

bcremer committed:
Do not test sys_tmp_dir if cache dir is provied. Fix #65.

Signed-off-by: Benjamin Cremer <benjamin.cremer@check24.de>

Ocramius commented on .laminas-ci.json:
```diff
@@ -1,2 +1,12 @@
 {
+  "additional_checks": [
```
Instead of adding a whole CI run, can we have a single test that uses `exec` on a dummy script, perhaps? 🤔

boesing replied:
Do you expect that test to check the exit code or how do we ensure that exactly the non-writable directory leads to the issue and not something else?

boesing replied:
I moved the check into a dedicated script which is now executed from a test method.

Ocramius reviewed (approved):
LGTM meanwhile 👍

boesing committed:
qa: add macos temp directory support

Signed-off-by: Maximilian Bösing <2189546+boesing@users.noreply.github.com>

boesing committed:
qa: move test into dedicated script which is executed from unit test

This reduces the number of Docker containers spun up during CI and results in fewer total checks.

Signed-off-by: Maximilian Bösing <2189546+boesing@users.noreply.github.com>

boesing commented:
@bcremer FYI, it seems your GPG key is not stored on github and thus your commit is marked "unverified"
